.travis.yml

```diff
@@ -26,6 +26,8 @@ env:
     - MIX_ENV: test
     - DATABASE_URL: postgres://elixir_boilerplate:password@localhost:5433/elixir_boilerplate_test
     - PGPORT: 5433
+    - SECRET_KEY_BASE: yv8eF+KYEbap8J8I9h1bLtWliN/tF6uFa5ba+/pxrTmDYf9kCUMSpb1R2RYTKQ8V 
+    - SIGNING_SALT: 2ly5tiMCHPachCPOIQjMcrk6PxRaXA25QFpaspnuumMyJZgAyhH8O1d59FluvwyS
```

Comment:
Oh… this is a drawback from having `config/test.exs` (which sets these values) **before** `rel/config/runtime.exs` 😬 

I think we have to find a way to avoid doing this.